packages/integration1d/integration1d.0.5.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+authors: ["Christophe Troestler <Christophe.Troestler@umons.ac.be>"]
+tags: ["science" "numerics"]
+license: "ISC"
+homepage: "https://github.com/Chris00/integration1d"
+dev-repo: "git+https://github.com/Chris00/integration1d.git"
+bug-reports: "https://github.com/Chris00/integration1d/issues"
+doc: "https://Chris00.github.io/integration1d/doc"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03"}
```

Comment:
```suggestion
  "ocaml" {>= "4.07"}
```
This package fails with OCaml < 4.07:
```

#=== ERROR while compiling integration1d.0.5.1 ================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.06.1 | pinned(https://github.com/Chris00/integration1d/releases/download/0.5.1/integration1d-0.5.1.tbz)
# path                 ~/.opam/4.06/.opam-switch/build/integration1d.0.5.1
# command              ~/.opam/4.06/bin/dune build -p integration1d -j 2
# exit-code            1
# env-file             ~/.opam/log/integration1d-15-6e09d5.env
# output-file          ~/.opam/log/integration1d-15-6e09d5.out
### output ###
#     ocamldep src/.integration1d.objs/integration1D.pp.ml.d (exit 2)
# (cd _build/default && /home/opam/.opam/4.06/bin/ocamldep.opt -modules -impl src/integration1D.pp.ml) > _build/default/src/.integration1d.objs/integration1D.pp.ml.d
# File "src/integration1D.ml", line 574, characters 0-2:
# Error: Syntax error
```

Reply:
Fixed the code (`;; ;;` was not accepted — there was a macro in between).